packages/opencensus-core/src/stats/view.ts

```diff
@@ -187,9 +196,58 @@ export class BaseView implements View {
     }
   }
 
+  /**
+   * Gets view`s metric
+   * @returns {Metric}
+   */
+  getMetric(): Metric {
+    const {type} = this.metricDescriptor;
+    let startTimestamp: Timestamp;
+
+    switch (type) {
+      case MetricDescriptorType.GAUGE_INT64:
+      case MetricDescriptorType.GAUGE_DOUBLE:
+        startTimestamp = null;
+        break;
+      default:
+        const [seconds, nanos] = process.hrtime();
+        startTimestamp = {seconds, nanos};
+    }
+
+    const timeseries: TimeSeries[] = [];
+
+    Object.keys(this.rows).forEach(key => {
+      const {tags} = this.rows[key];
+      const labelValues: LabelValue[] = MetricUtils.tagsToLabelValues(tags);
+      const point: Point = this.toPoint(startTimestamp, this.getSnapshot(tags));
+      timeseries.push({startTimestamp, labelValues, points: [point]});
+    });
+
+    return {descriptor: this.metricDescriptor, timeseries};
+  }
+
+  /**
+   * Converts snapshot to point
+   * @param timestamp The timestamp
+   * @param data The aggregated data
+   * @returns {Point}
+   */
+  private toPoint(timestamp: Timestamp, data: AggregationData): Point {
+    let value;
+    if (data.type === AggregationType.DISTRIBUTION) {
```

Comment:
Point is missing from other AggregationData like ```SumData```, ```CountData``` and ```LastValueData```.

Reply:
@mayurkale22 I remember that we discussed that I dont need to implement everything and some things will be added in future
/cc @jparsana

Reply:
This is part of the basic requirement "**convert Stats data models to Metric data models**".

Reply:
@mayurkale22 could you please provide more details - what should be done by https://github.com/census-instrumentation/opencensus-node/issues/213

Reply:
Done